RealmSwift/Object.swift

```diff
@@ -419,6 +419,16 @@ public class ObjectUtil: NSObject {
         } as NSArray
     }
 
+    // Return the indices of generic properties in the object.
+    @objc private class func getGenericListPropertyIndices(_ object: Any) -> NSArray {
+        let children = Mirror(reflecting: object).children
+        let indices = Array(0..<Int(children.count))
```

Comment:
Need to verify that `children` actually corresponds to properties on the object, and that (e.g.) child number 0 is actually the first property etc